primitives/binary_operators.futil

```diff
@@ -86,8 +86,7 @@ extern "binary_operators.sv" {
   /// =================== Unsigned, Bitnum =========================
   /// Other unsigned bitnum primitives are found in the core library,
   /// since they're required for FSM encoding.
-
-  primitive std_mult_pipe<"state_share"=1>[WIDTH](
+  primitive std_mult_seq<"state_share"=1>[WIDTH](
```

Comment:
Let's keep the original name and add a comment to #1596 to rename this primitive when we update the library.

Reply:
Yep I agree here.